camel/models/anthropic_model.py

```diff
@@ -34,8 +34,9 @@ class AnthropicModel(BaseModelBackend):
     Args:
         model_type (ModelType): Model for which a backend is created, one of
             CLAUDE_* series.
-        model_config_dict (Dict[str, Any]): A dictionary that will be fed into
-            Anthropic.messages.create().
+        model_config_dict (Optional[Dict[str, Any]], optional): A dictionary
+            that will be fed into Anthropic.messages.create(). (default:
+            :obj:`AnthropicConfig().as_dict()`)
```

Comment:
Kind of unsure about this, which one do you think is better

```
If None, then AnthropicConfig().as_dict() will be used. (default: None)
```

```
(default: AnthropicConfig().as_dict())
```

Reply:
Thanks @WHALEEYE , the first one is better